aries/aries_vcx_core/src/wallet/base_wallet.rs

```diff
@@ -1,124 +1,440 @@
-use std::collections::HashMap;
-
 use async_trait::async_trait;
+use derive_builder::Builder;
```

Comment:
Ah, I see that `derive_builder` was in fact added in this PR. I've left a comment about it somewhere above.